src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -927,6 +910,7 @@
         <choice>
             <group>
                 <ref name="h1"/> <!-- h1 -->
+                <ref name="h2"/> <!-- h2 -->
```

Comment:
This will require both a h1 and a h2. Use a `choice` to require either or:

```xml
<choice>
    <group>
        <choice>
            <ref name="h1"/> <!-- h1 -->
            <ref name="h2"/> <!-- h1 -->
        </choice>
        <oneOrMore>
```
```